manifests/satellite6.pp

```diff
@@ -0,0 +1,19 @@
+# Class: satellite6
+#   Install the SAtellite 6 flavor of the robottelo_slave
+#
+class robottelo_slave::satellite6 {
+  include ::robottelo_slave
+
+  ensure_packages([
+    'brewkoji',
+  ])
+
+  $rh_mirror='http://download.lab.bos.redhat.com'
```

Comment:
if this is internal server, we need to remove from here and perhaps store it in jenkins.